docs/examples/workflow_and_job/hourly_workflow.py

```diff
@@ -14,8 +15,8 @@ def run(self, runtime):
 
 hourly_workflow = Workflow(
     workflow_id='hourly_workflow',
-    runtime_as_datetime=True,
     schedule_interval='@hourly',
+    start_time_expression_factory=hourly_start_time,
```

Comment:
why factory for a string literal ?

Reply:
what's `start_time_expression_factory` after all?

Reply:
We need this expression to set the start time for each of our workflows. There is no general rule when to start a daily workflow or hourly workflow (or any other type).

A example factory looks like this:
```def hourly_start_time(start_time: str) -> str:
    return f'datetime.strptime("{start_time}", "%Y-%m-%d %H:%M:%S") - (timedelta(seconds={get_timezone_offset_seconds()}))'```

But I think I can make it a string.